testsuite/tests/basic/boxedints.ml

```diff
@@ -1,4 +1,6 @@
-(* TEST *)
+(* TEST
+   flags = "-w -3"  (* silence deprecation warnings *)
```

Comment:
Because this file is testing functions (`Int32.format` and `Int64.format`) and it does so through a functor, which somehow means that adding `[@@@ocaml.warnings "-3"]` at the top of the file isn't enough to silence the warning.

Reply:
This sounds like a bug (warning attribute not taken into account), no?  (Note that the attribute is called ocaml.warning without the s.)

Reply:
You're right, it works much better with the correct syntax.